pkg/machinehealthcheck/machinehealthcheck.go

```diff
@@ -68,57 +49,105 @@ var _ = Describe("[Feature:MachineHealthCheck] MachineHealthCheck", func() {
 	})
 
 	AfterEach(func() {
+		By("Deleting the MachineHealthCheck resource")
+		err := client.Delete(context.Background(), machinehealthcheck)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
I would have inlined this, but as it matches the existing, can leave it (or update both) `Expect(client.Delete(...)).To(Succeed()`